README.md

```diff
@@ -1,62 +1,103 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Motorcycle-booking-service: Back-end
 
-> Description the project.
+> An API that provides endpoints for motorcycle-booking-service-front-end
 
+## Kanban Board
+[Kanban board link](https://github.com/sentayhu19/Book-an-appointment-back-end/projects/1)
 
-## Built With
+Kanban board screenshot![kanban](https://user-images.githubusercontent.com/77745775/184105254-444d5425-98de-4e7e-b920-79e09d1032ea.png)
+
+There are 4 members in this Group
+
+1. Solomon Hagan @Achilles-Dev 
+2. Debas Gebreslasie @Debas-31 
+3. Ayoub Chahir @AyoubMs 
+4. Sentayhu Berhanu @sentayhu19 
 
-- Major languages
-- Frameworks
-- Technologies used
 
-## Live Demo (if available)
+## Built With
+
+- Ruby
+- Ruby on Rails
+  
+ ## Live Demo
+ ### [Front-end link](https://github.com/AyoubMs/book-an-appointment-front-end)
+ 
+ 
+ - [API-docs](http://127.0.0.1:3000/api-docs/index.html)
```

Comment:
[OPTIONAL]
The layout of this section is kind of wonky, also, you should probably put actual demo links here. The links are kind of misleading since it's under the "Live Demo" section